feetech_ros2_driver:
  ros__parameters:
    node:
      frequency: 100

    driver:
      port_name: "/dev/ttyUSB0"
      baud_rate: 1000000
      frequency: 100
      continuous_positions: true

    servos:
      ids: [1, 2, 3, 4, 5, 6, 7, 8]
      homing_modes: [0, 0, 0, 0, 0, 0, 0, 0]
      max_speeds: [100, 100, 100, 100, 100, 100, 100, 100]
      max_currents: [100, 100, 100, 100, 100, 100, 100, 100]
